include/picongpu/plugins/saxs/jupyter/validate.ipynb

```diff
@@ -0,0 +1,4975 @@
+{
```

Comment:
Please export the notebook without temporary data. There is an option to clear the state of all cells.